express-static/index.js

```diff
@@ -0,0 +1,13 @@
+const express = require('express');
+const path = require('path');
+const app = express();
+const staticFiles = express.static('public');
+
+path.join(__dirname, 'public');
```

Comment:
The return value of `path.join()` should be used as the argument to `express.static()`